src/test/java/seedu/address/logic/commands/person/DeletePersonCommandTest.java

```diff
@@ -102,7 +102,7 @@ public void execute_validLessonCodeUnfilteredList_success() {
         Set<LessonCode> lessonCodeToDelete = new HashSet<>();
         lessonCodeToDelete.add(new LessonCode("T12"));
         DeletePersonCommand deletePersonCommand = new DeletePersonCommand(new ModuleCodesContainsKeywordsPredicate(
-                Arrays.asList(String.format("[%s]", VALID_MODULE_CODE_CS2100 + " T12"))),
+                Arrays.asList(String.format("%s", VALID_MODULE_CODE_CS2100 + " T12"))),
```

Comment:
```suggestion
                Arrays.asList(String.format("%s", VALID_MODULE_CODE_CS2100))),
```
you can delete all the classes behind in the ModuleCodeContainsKeywordsPredicate.

Reply:
Thanks!